datasets/sipecam.yaml

```diff
@@ -0,0 +1,21 @@
+Name: SiPeCaM (Sitios Permanentes de la Calibración y Monitoreo de la Biodiversidad)
+Description: The SiPeCaM goal is to create a data source that allows to evaluate changes in the biodiversity state, considering key aspect of how does the ecosystem behaves.
+Documentation: https://github.com/CONABIO/sipecam-open-data/blob/main/docs/SiPeCaM.md
+Contact: "Michael F. Schmidt [mschmidt@conabio.gob.mx](mschmidt@conabio.gob.mx)"
+ManagedBy: "[Conabio](https://monitoreo.conabio.gob.mx/)"
+UpdateFrequency: SiPeCaM offers data releases once the biologist recollect the data from the protected natural areas (every 3 months)
+Tags:
+  - biodiversity
+  - fauna
+  - animals audio
+  - protected areas
+License: Creative Commons with attribution
```

Comment:
Could you link to an actual license. I think in this case you are looking for https://creativecommons.org/licenses/by/4.0/ ?